nginx.conf

```diff
@@ -40,20 +40,23 @@ http {
 
     # Prod
     server {
-       listen 8080;
-       root /usr/share/nginx/auth;
+        listen 8080;
+        root /usr/share/nginx/auth;
 
         # Health endpoint for k8s
-       location = /health {
+        location = /health {
```

Comment:
future improvement: lets move this endpoint elsewhere!

Reply:
Where do you think we should move it? Another port?

Reply:
> Another port?

Yeah probably. and/or find some sort of metrics plugin